rtsp.go

```diff
@@ -215,16 +247,91 @@ func (rc *rtspCamera) reconnectClient() (err error) {
 			}
 		}
 	})
-	_, err = rc.client.Play(nil)
+
+	return nil
+}
+
+// initH265 initializes the H265 decoder and sets up the client to receive H265 packets.
+func (rc *rtspCamera) initH265(tracks media.Medias, baseURL *url.URL) (err error) {
+	var format *formats.H265
+
+	track := tracks.FindFormat(&format)
+	if track == nil {
+		rc.logger.Warn("tracks available")
+		for _, x := range tracks {
+			rc.logger.Warnf("\t %v", x)
+		}
+		return errors.New("h265 track not found")
+	}
+
+	_, err = rc.client.Setup(track, baseURL, 0, 0)
 	if err != nil {
 		return err
 	}
-	clientSuccessful = true
+
+	rtpDec, err := format.CreateDecoder2()
+	if err != nil {
+		rc.logger.Errorf("error creating H265 decoder %v", err)
+		return err
+	}
+
+	rc.rawDecoder, err = newH265Decoder()
+	if err != nil {
+		return err
+	}
+
+	// For H.265, handle VPS, SPS, and PPS
+	if format.VPS != nil {
+		rc.rawDecoder.decode(format.VPS)
+	} else {
+		rc.logger.Warn("no VPS found in H265 format")
+	}
+
+	if format.SPS != nil {
```

Comment:
Can h265 image decoding succeed if VPS, SPS or PPS are not set?

Reply:
I need to test more thoroughly here. But, my understanding is that is possible for ffmpeg hevc decoder to work properly without VPS if SPS and PPS are available in certain circumstances.